tools/k8s/README.md

```diff
@@ -18,18 +18,7 @@ on some cloud machine.
 If you go the GKE cloude route, keep in mind that some GKE configurations
 create an API server that is not generally accessible.
 
-## Step 2: Install a suitable service account to your cluster
-
-Read `tools/k8s/my-service-account.yaml` so you know what it's doing, then apply
-it:
-
-```sh
-kubectl apply -f tools/k8s/my-service-account.yaml
-```
-
-This step is only required once after creating your cluster.
-
-## Step 3: Run `determined-master` with a special `devcluster.yaml`
+## Step 2: Run `determined-master` with a special `devcluster.yaml`
```

Comment:
Before this step, should we include the instructions for running `gcloud container clusters get-credentials`  to pull the cluster creds into the kubeconfig before running `devcluster`?

Reply:
I can add a blurb to make sure your kubeconfig is correct, I kinda took that as a given.

I'll probably add a minikube example, but it'll be different depending on gke/minikube/kind/etc.